package.json

```diff
@@ -20,7 +20,7 @@
     "docs": "gulp docs",
     "api": "gulp api-docs"
   },
-  "version": "6.0.0-rc.5",
+  "version": "6.0.0-rc.11",
```

Comment:
Why is it going from rc.5 to rc.11?

Reply:
My guess is because this was published [to NPM](https://www.npmjs.com/package/@angular/material) as rc.11 already, and this is an after-the-fact update

Reply:
Yep; it's not yet possible to test `ng update` without publishing to npm